### PR TITLE
upgrader: UI improvements

### PR DESCRIPTION
**Pull request by mlyle**
Hopefully this is a bit clearer.  Still don't like some aspects (the text changing back "early", the button placement), but... this is a good first step improvement.

connects to #821

---

This change is [Reviewable](https://reviewable.io/reviews/d-ronin/dronin/837)

### CODE REVIEW

**Comment by mlyle** [comment]
What it looks like: 

https://www.youtube.com/watch?v=9dnkJ2NPjxY

Note the file dialog opens bigger than the portion of the screen I'm recording at one point.  Still was all on screen / not a bug.

**Comment by mlyle** [comment]
tested by @droidicus, he says it's good.

**Comment by mlyle** [comment]
Worked for me in a few cycles of testing.

**Comment by mlyle** [comment]
@pug398 tested a branch with this and likes the results

File: ground/gcs/src/plugins/uploader/uploadergadgetwidget.cpp [hunk 1]
```diff
@@ -1108,17 +1128,24 @@ void UploaderGadgetWidget::doUpgradeOperation()
...
-    }
+            return;
+        }
 
-    /* trigger import of saved settings. */
-    if (!importMngr->importUAVSettings(xmlDump)) {
-        upgradeError(tr("Unable to import UAV settings from upgrade!"));
+        ret = m_dialog.PromptUser(
+                tr("Please review the imported settings and save to board."),
+                tr("No settings are currently saved.  Select the "
```
Comment (mluessi) [inline ground/gcs/src/plugins/uploader/uploadergadgetwidget.cpp]: spurious space after .

**Comment by mluessi** [comment]
LGTM otherwise

File: ground/gcs/src/plugins/uploader/uploadergadgetwidget.cpp (hunk 1 above)
Comment (mlyle) [inline ground/gcs/src/plugins/uploader/uploadergadgetwidget.cpp]: After the dot?  I was taught to double space after a period..

**Pull request merged**